obal/__init__.py

```diff
@@ -164,41 +164,53 @@ def __lt__(self, other):
         return NotImplemented
 
 
-def find_playbooks(playbooks_path):
+class Layout(object):
     """
-    Find all playbooks in the given path.
+    A class describing the where to find various files
```

Comment:
So one thing why I called this `Config` on IRC was: this class could also contain the CLI name of the tool to be passed in

```python
parser = argparse.ArgumentParser('obal')
```

Reply:
That's a good argument. Now I'm thinking about `Application`, but that would imply there's much more logic to it. How about `ApplicationConfig`?

Reply:
Sold.

Reply:
Updated